src/extensions/default/OpenWithExternalApplication/node/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "brackets-open-external_application",
+  "dependencies": {
+    "open": "0.0.5"
```

Comment:
open module does not seem to be latest.
Quoting https://www.npmjs.com/package/open
```
Note: The original open package was previously deprecated in favor of this package, and we got the name, so this package is now named open instead of opn. If you're upgrading from the original open package (open@0.0.5 or lower), keep in mind that the API is different.
```

Reply:
above 0.0.5 , open module version starts with 6.0.0. Open module starting 6.0.0 is not compatible with node version 6.11.0.

It gives below error  when requiring module:
 const wslToWindowsPath = async path => {
                               ^^^^

SyntaxError: Unexpected identifier
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)